plugins/keycloak-backend/README.md

```diff
@@ -54,33 +61,43 @@ catalog:
         clientSecret: ${KEYCLOAK_CLIENTSECRET}
 ```
 
+Optionally, you can override the default parameters by adding the basic configuration to `app-config.yaml`.
+
+```yaml
+keycloak:
+  userQuerySize: 500
+  groupQuerySize: 250
+```
+
 Finally, register the plugin in the `packages/backend/src/plugins/catalog.ts` file.
 
 ```javascript
  // packages/backend/src/plugins/catalog.ts
-+import { KeycloakOrgEntityProvider } from '@janus-idp/backstage-plugin-keycloak-backend';
-
- export default async function createPlugin(
-   env: PluginEnvironment,
- ): Promise<Router> {
-   const builder = await CatalogBuilder.create(env);
-
-+  builder.addEntityProvider(
-+    KeycloakOrgEntityProvider.fromConfig(env.config, {
-+      id: 'development',
-+      logger: env.logger,
-+      schedule: env.scheduler.createScheduledTaskRunner({
-+        frequency: { hours: 1 },
-+        timeout: { minutes: 50 },
-+        initialDelay: { seconds: 15 }
-+      }),
-+    }),
-+  );
+import { KeycloakOrgEntityProvider } from '@janus-idp/backstage-plugin-keycloak-backend';
+
+export default async function createPlugin(
+  env: PluginEnvironment,
+): Promise<Router> {
+  const builder = await CatalogBuilder.create(env);
+
+  builder.addEntityProvider(
+    KeycloakOrgEntityProvider.fromConfig(env.config, {
+      id: 'development',
+      logger: env.logger,
+      userQuerySize: env.config.getOptionalNumber('keycloak.userQuerySize'),
+      groupQuerySize: env.config.getOptionalNumber('keycloak.groupQuerySize'),
```

Comment:
why is this not consumed from `env.config` within the provider and is passed here separately? The config object is already available to the provider via `KeycloakOrgEntityProvider.fromConfig(env.config, ...`